README.md

```diff
@@ -25,8 +26,31 @@ the API. For more details, see https://github.com/evergreen-ci/evergreen/wiki/RE
 
 This is a Python client library for interacting with Evergreen and Evergreen objects. It currently only
 supports the V2 version of Evergreen's api. It can be used either by Python code in a separate application
-or on the command line to get data about Evergreen objects quickly and easily. 
+or on the command line to get data about Evergreen objects quickly and easily.
 
+## Getting Help
+
+**What's the right channel to ask my question?**
```

Comment:
[nit] You might want to include a blank line between these headers and the responses or else markdown will join them on the same line. (This applies to all the sections in the update).

Also, you might consider using a subheader (`###`) instead of just bolding, since it seems like it would fit here.